src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -1467,6 +1468,11 @@ struct MHAHelper {
         _weight.resize<float>({size_t{1}, size_t{1}, size_t{1}, size_t{1}});
     }
 
+    void resize_temporary_weight_buffer(const size_t& h) {
```

Comment:
NIT. There is no need to pass "size_t" by const reference, it's better to pass it by value.